packages/core-react/index.d.ts

```diff
@@ -8,6 +8,15 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
+export type RadioProps = {
+  className?: string,
+  id: string,
```

Comment:
I think maybe id should be optional

Reply:
Ah ok, I was on the edge about that too

Reply:
I think each line should be finished by ; not ,